doc/modules/mod_event_pusher_rabbit.md

```diff
@@ -0,0 +1,198 @@
+  * `sent_topic` (string, default: `<<"groupchat_msg_sent">>`) - Defines RabbitMQ group chat message sent topic name;
+  * `recv_topic` (string, default: `<<"groupchat_msg_recv">>`) - Defines RabbitMQ group chat message received topic name;
+
+### Example configuration
+
+```Erlang
+{mod_event_pusher, [
+    {backends, [
+        {rabbit, [
+            {presence_exchange, [{name, <<"presence">>},
+                                 {type, <<"topic">>}]},
+            {chat_msg_exchange, [{name, <<"chat_msg">>},
+                                 {sent_topic, <<"chat_msg_sent">>},
+                                 {recv_topic, <<"chat_msg_recv">>}]},
+            {groupchat_msg_exchange, [{name, <<"groupchat_msg">>},
+                                      {sent_topic, <<"groupchat_msg_sent">>},
+                                      {recv_topic, <<"groupchat_msg_recv">>}]}
+        ]}
+    ]}
+]}
+```
+
+### JSON Schema examples
+The different kinds of notifications deliver slightly different messages.
+The messages are delivered in a JSON format.
+#### Presence updates
+
+The JSON format for an online presence update notification is:
+```JSON
+{
+    "user_id": "alice@localhost/res1",
+    "present": true
+}
+```
+
+For offline presence updates, the `present` boolean value is set to false:
+
+```JSON
+{
+    "user_id": "alice@localhost/res1",
+    "present": false
+}
+```
+#### Sent/received messages
+The JSON format for a private message notification is:
+```JSON
+{
+    "to_user_id": "bob@localhost/res1",
+    "message": "Hello, Bob",
+    "from_user_id": "alice@localhost/res1"
+}
+```
+The notification is similar for group messages. For example for "sent" events:
+```JSON
+{
+    "to_user_id": "muc_publish@muc.localhost",
+    "message": "Hi, Everyone!",
+    "from_user_id": "bob@localhost/res1"
+}
+```
+and for "received" events:
+
+```JSON
+{
+    "to_user_id": "bob@localhost/res1",
+    "message": "Hi, Everyone!",
+    "from_user_id": "muc_publish@muc.localhost/alice"
+}
+```
+
+### Metrics
+
+The module provides some metrics related to RabbitMQ connections and messages
+as well. Provided metrics:
+
+  * `connections_active` - number of active connections to a RabbitMQ
+  server
+  * `connections_opened` - number of opened connections to a RabbitMQ
+  server since module startup
+  * `connections_closed` - number of closed connections to a RabbitMQ
+  server since module startup
+  * `connections_failed` - number of failed connections to a RabbitMQ
+  server since module startup
+  * `messages_published` - number of published messages to a RabbitMQ server
+  since module startup
+  * `messages_timeout` - number of messages to a RabbitMQ server that weren't
+  confirmed by the server since module startup
+  * `messages_failed` - number of messages to a RabbitMQ server that were
+  rejected by the server since module startup
+  * `message_publish_time` - amount of time it takes to publish a message to
```

Comment:
Lacks information whether time is measured for all publishes or only successful ones.